services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -78,6 +78,9 @@ def _get_environment_variables(
         "S3_SECRET_KEY": r_clone_settings.R_CLONE_S3.S3_SECRET_KEY,
         "S3_SECURE": f"{r_clone_settings.R_CLONE_S3.S3_SECURE}",
         "SC_BOOT_MODE": f"{app_settings.DYNAMIC_SERVICES.DYNAMIC_SIDECAR.DYNAMIC_SIDECAR_SC_BOOT_MODE}",
+        "SSL_CERT_FILE": f"{app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME}"
+        if app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME
+        else None,  # via https://stackoverflow.com/questions/31448854/how-to-force-requests-use-the-certificates-on-my-ubuntu-system#comment78596389_37447847# via https://stackoverflow.com/questions/31448854/how-to-force-requests-use-the-certificates-on-my-ubuntu-system#comment78596389_37447847
```

Comment:
if you where to provide some documentation. I think it makes more sense to document this in the settings.

Reply:
It should also go here imho because it explains the ambiguity in why the specific env-var  `SSL_CERT_FILE` is assigned.

Reply:
then please put the comments on two separate lines above so they are more easily read